Source/Core/Core/NetPlayServer.cpp

```diff
@@ -849,14 +924,318 @@ bool NetPlayServer::StartGame()
   spac << m_settings.m_EXIDevice[1];
   spac << static_cast<u32>(g_netplay_initial_rtc);
   spac << static_cast<u32>(g_netplay_initial_rtc >> 32);
+  spac << m_settings.m_SyncSaveData;
+  spac << region;
 
   SendAsyncToClients(std::move(spac));
 
+  m_start_pending = false;
   m_is_running = true;
 
   return true;
 }
 
+// called from ---GUI--- thread
+bool NetPlayServer::SyncSaveData()
+{
+  m_save_data_synced_players = 0;
+
+  u8 save_count = 0;
+  for (size_t i = 0; i < 2; i++)
```

Comment:
This magic `2` keeps appearing here and further down, I wonder if you can write those pieces of code differently (like, by using an array of interresting values/references, or at least by making it a named `constexpr` to give some context)

Reply:
It's a hardware constant (number of memory card slots) so I don't really see an issue with it.

Reply:
I never intended for all occurrences to be replaced; just a local that gives it a useful name (so your future self and other devs reading the code can distinguish it from other places where 2 means something else). Like `constexpr size_t max_exi_device_count = 2;` or something similar that captures the meaning.
But yeah, I also get the silly points; so feel free to ignore this one.

Reply:
In this case, I believe NetSettings' `m_EXIDevice` member in NetPlayProto.h could be made into a `std::array` and then the condition could just be:

```cpp
for (std::size_t i = 0; i < m_settings.m_EXIDevice.size(); i++)
{
  // Code here ...
}
```

I didn't bring it up initially in review, because I'm planning on converting the `TEXIDevices` enum into an enum class after this, which would also involve converting to `std::array` where applicable